### ISSUE TITLE
can one use dicom images for training instead of jpg

### ISSUE THREAD

**Issue by RobsoUganda** [chaoyi-wu/RadFM#37]
hello Author, thanks for the work. can one use dicom(.dcm) images for training instead of jpg ?

**Comment by hari3100** [comment]
I have the same query , or can we convert dicom to jpg image slices and use those multiple jpg images together in a single inference?
@xiaoman-zhang  thank you

**Comment by chaoyi-wu** [comment]
Certainly, for 3D cases we use nifty by default and for 2D cases we use jpg images.
